[DOCS-14078] Update Host List for new UI (#36463)
DataDog/documentation: content/en/infrastructure/list.md

## Overview

> The Host List gives you a live inventory of all hosts reporting to Datadog through the Agent or cloud integrations. By default, it shows hosts with activity in the last two hours, but you can extend the view to cover up to one week. To open the Host List, navigate to [**Infrastructure > Hosts**][10] in Datadog. Don't use this list to estimate your infrastructure host billing. See the [billing][11] page to learn about billing.

Author: ```suggestion
The Host List gives you a live inventory of all hosts reporting to Datadog through the Agent or cloud integrations. By default, it shows hosts with activity in the last two hours, but you can extend the view to cover up to one week. To open the Host List, navigate to [**Infrastructure > Hosts**][10] in Datadog. This list should not be used to estimate your infrastructure host billing. See the [billing][11] page to learn about billing.
```
